src/applications/differential/conduit/DifferentialCreateCommentConduitAPIMethod.php

```diff
@@ -114,4 +123,49 @@ protected function execute(ConduitAPIRequest $request) {
     );
   }
 
+  public static function loadUnsubmittedInlineComments(
+    PhabricatorUser $viewer,
+    DifferentialRevision $revision) {
+
+    $inlines = id(new DifferentialDiffInlineCommentQuery())
+      ->setViewer($viewer)
+      ->withRevisionPHIDs(array($revision->getPHID()))
+      ->withAuthorPHIDs(array($viewer->getPHID()))
+      ->withHasTransaction(false)
+      ->withIsDeleted(false)
+      ->needReplyToComments(true)
+      ->execute();
+
+    foreach ($inlines as $key => $inline) {
+      $inlines[$key] = DifferentialInlineComment::newFromModernComment(
+        $inline);
+    }
+
+    PhabricatorInlineComment::loadAndAttachVersionedDrafts(
+      $viewer,
+      $inlines);
+
+    // Don't count void inlines when considering draft state.
+    foreach ($inlines as $key => $inline) {
+      if ($inline->isVoidComment($viewer)) {
+        unset($inlines[$key]);
+        continue;
+      }
+
+      // For other inlines: if they have a nonempty draft state, set their
+      // content to the draft state content. We want to submit the comment
+      // as it is currently shown to the user, not as it was stored the last
+      // time they clicked "Save".
+
+      // $draft_content = $inline->getContentForEdit($viewer);
+      // if (strlen($draft_content)) {
+      //   $inline->setContent($draft_content);
+      // }
```

Comment:
nit: Was this copied from elsewhere? Might make sense to remove this altogether instead of re-adding it as commented-out code.

Reply:
Yes, it was copied from: https://github.com/mozilla-conduit/phabricator/blob/397648855f8bc272db1876da8ce2254d3da82fb9/src/applications/differential/query/DifferentialTransactionQuery.php#L10-L53

Reply:
> Might make sense to remove this altogether instead of re-adding it as commented-out code.

I kept the commented-out code to have a reference for future use, if needed.

Reply:
Ok. If I understand correctly, from an API perspective it is not relevant since it only applies to unsaved draft comments on the UI? Not sure we need to introduce commented out code, even if it might be used in the future. Perhaps a comment explaining that additional lines were removed that were no longer needed?